openredir-cli

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+from argparse import ArgumentParser
+from tempfile import TemporaryDirectory
+
+
+def main():
+    parser = ArgumentParser(description="openredir CLI helper")
+    parser.add_argument(
+        '-r', '--rule',
+        metavar="RULE",
+        action="append",
+        help='Specify a rule in ORIGIN:REPLACEMENT format, can be specified multiple times. (Example: /etc/resolv.conf:/tmp/my_resolv.conf)',
+    )
+    parser.add_argument(
+        'command',
+        metavar="COMMAND",
+        nargs="+",
+        help="Command to run")
+
+    options = parser.parse_args()
+
+    LUA_SRC = "function redirect(f, s)"
+
+    for rule in options.rule:
+        LUA_SRC += 's = s:gsub("{}", "{}")'.format(*rule.split(":"))
```

Comment:
'"' needs to be escaped. Also the replacement should happen at the start (if only absolution path is supported).

Reply:
Actually I think I can do Lua pattern match here? So perhaps escaping is not really mandatory... I also want to support partial names if possible.

Reply:
Ahhh yes. It's still needed for the user to learn Lua :-)